manifest.json

```diff
@@ -1814,6 +1814,11 @@
         {
             "js": ["plugins/celebs-place.js"],
             "matches": ["*://*.celebs-place.com/*"]
+        },
+        {
+            "js": ["plugins/viewbug.js"],
+            "matches": ["*://*.viewbug.com/*"],
+            "all_frames": true
```

Comment:
@GrosPoulet Does this site _actually_ use iframes?

Reply:
Yes, some search results are displayed in an iframe.